vertx-web/src/main/java/io/vertx/ext/web/impl/Utils.java

```diff
@@ -331,13 +335,45 @@ public static Buffer readResourceToBuffer(String resource) {
     }
   }
 
+  private static final Charset DEFAULT_CHARSET = Charset.defaultCharset();
+
   /*
-  Reads from file or classpath
+  Reads from file or classpath using the default charset, replacing CRLF line endings with LF ones
```

Comment:
This will be a behavior breaking change and I'm not sure it is correct. One can use the readFileToString to load anything, including, say for example, `yaml`, `json`, etc... where in most cases replacing line endings does not introduce side effects, but it could do with some formats.